openedx/core/djangoapps/oauth_dispatch/tests/mixins.py

```diff
@@ -63,6 +63,7 @@ def _decode_jwt(verify_expiration):
             'scopes': scopes,
             'version': settings.JWT_AUTH['JWT_SUPPORTED_VERSION'],
             'sub': anonymous_id_for_user(user, None),
+            'email_verified': user.is_active,
```

Comment:
I like having names that make sense, but will we ever care about size? Is there a reason the others are so short?
Not a blocker, just a question.

Reply:
Some of these names, including `aud`, `iat`, `exp`, `iss`, `sub` and `preferred_username` come from the OpenID standard: [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) and [Standard Claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).  `email_verified` is also included as a Standard Claim.  I'm not sure why they choose to abbreviate some and not others.

Reply:
Interesting. Thanks.